rlImGui/rlImGui.cs

```diff
@@ -259,6 +282,9 @@ unsafe internal static void rlImGuiSetClipText(IntPtr userData, sbyte* text)
         private unsafe delegate sbyte* GetClipTextCallback(IntPtr userData);
         private unsafe delegate void SetClipTextCallback(IntPtr userData, sbyte* text);
 
+        private static GetClipTextCallback getClipCallback = null!;
+        private static SetClipTextCallback setClipCallback = null!;
+
```

Comment:
Is this all that's needed for the crash fix?

Reply:
Yes. I did make a test with the old code, my code, and a GC button. On the old code, the clipboard would work fine until I pressed the GC button, at which point copying or pasting from/to a text field would cause a hard crash on the program. With my new code, the clipboard still worked fine even after pressing the GC button.